tests/test_transport.py

```diff
@@ -77,12 +77,12 @@ async def handle_async_request(request: Request) -> Response:
         yield mock_asleep, status_code_sequences
 
 
-class MockHTTPTransport(httpx.HTTPTransport):
+class MockHTTPTransport(httpx.BaseTransport):
```

Comment:
Could we remove the `HTTP` from these transport names, as they are no longer anything to do with http